docs/web/user_guide.md

```diff
@@ -637,11 +637,39 @@ filter arguments:
                         only if basename or newname is a run name (on the
                         remote server).
   --detected-at TIMESTAMP
-                        Filter results by detection date. The format of
-                        TIMESTAMP is 'year:month:day:hour:minute:second' (the
-                        "time" part can be omitted, in which case midnight
-                        (00:00:00) is used).
-  --fixed-at TIMESTAMP  Filter results by fix date. The format of TIMESTAMP is
+                        DEPRECATED. Use the '--detected-after' option to
+                        filter results by detection date. Filter results by
+                        detection date. The format of TIMESTAMP is
+                        'year:month:day:hour:minute:second' (the "time" part
+                        can be omitted, in which case midnight (00:00:00) is
+                        used).
+  --fixed-at TIMESTAMP  DEPRECATED. Use the '--detected-before' option to get
+                        the same behaviour. Filter results by fix date. The
+                        format of TIMESTAMP is
+                        'year:month:day:hour:minute:second' (the "time" part
+                        can be omitted, in which case midnight (00:00:00) is
+                        used).
+  --detected-before TIMESTAMP
+                        Get results which were detected before the given date.
```

Comment:
I think it would be nice to mention somehow that the detection date is the analysis date to be more explicit.

Reply:
This is not the analysis date but the storage date. See: https://github.com/Ericsson/codechecker/blob/master/web/server/codechecker_server/api/report_server.py#L2896

By the way I updated this help message and the user guide and mentioned it.